app/main.py

```diff
@@ -1,6 +1,21 @@
-# write your imports here
+from app.people.customer import Customer
+from app.people.cinema_staff import Cleaner
+from app.cinema.bar import CinemaBar
+from app.cinema.hall import CinemaHall
 
 
-def cinema_visit(customers: list, hall_number: int, cleaner: str, movie: str):
-    # write you code here
-    pass
+def cinema_visit(visitors: list,
+                 auditorium_number: int,
+                 cleaner_name: str,
+                 movie_title: str) -> None:
+    customers_list = [Customer(customer.get("name"), customer.get("food"))
+                      for customer in visitors]
+
+    cleaning_staff = Cleaner(cleaner_name)
+    hall = CinemaHall(auditorium_number)
+
+    for customer in customers_list:
+        product = customer.food
+        CinemaBar.sell_product(customer, product)
```

Comment:
There is no need for a `product` variable, you could pass it directly to the `sell_product`  
```suggestion
        CinemaBar.sell_product(customer, customer.food)
```

Reply:
як на мене цю змінну взагалі не обовязково передавати, доствтньо customer.  Вона тут тільки через умови завдання.